Tribler/Main/vwxGUI/CreditMiningPanel.py

```diff
@@ -0,0 +1,533 @@
+"""
+This module contains credit mining panel and list in wx
+
+Written by Ardhi Putra Pratama H
+"""
+
+import logging
+import wx
+from binascii import hexlify
```

Comment:
The import order is incorrect (there are three warnings raised by point because of these imports). Please use your IDE to reformat the imports.